src/less/success-message/base/success-message.less

```diff
@@ -40,6 +20,25 @@
     padding: 0 9%;
     text-align: center;
     width: 100%;
+
+    .btn--secondary,
```

Comment:
Is there a way we could maybe make this a stronger match. This will match any secondary button in the dialog. While I doubt there could be any others put in there maybe a stronger match will be better (like direct child of the main container which should only have a button)
This is not a big deal and more of a question.

Reply:
Hopefully they won't have any other buttons, but it won't hurt. 👍